pkgs/memory_usage/README.md

```diff
@@ -0,0 +1,23 @@
+[![pub package](https://img.shields.io/pub/v/leak_tracker.svg)](https://pub.dev/packages/leak_tracker)
```

Comment:
The things in this file which refer to package:leak_tracker (not the repo) should be updated to instead preference package:memory_usage (i.e., the pub.dev references).